src/test/kotlin/io/bazel/kotlin/builder/tasks/jvm/KotlinBuilderJvmJdepsTest.kt

```diff
@@ -38,71 +38,79 @@ class KotlinBuilderJvmJdepsTest(private val enableK2Compiler: Boolean) {
       return listOf(
         // TODO: Enable k2 for tests
 //        arrayOf(true),
-        arrayOf(false)
+        arrayOf(false),
       )
     }
   }
 
   val ctx = KotlinJvmTestBuilder()
 
   val TEST_FIXTURES_DEP = Dep.fromLabel(":JdepsParserTestFixtures")
+  val KOTLIN_STDLIB_DEP = Dep.fromLabel("//kotlin/compiler:kotlin-stdlib")
 
   @Test
   fun `no kotlin source produces empty jdeps`() {
-
     val deps = runJdepsCompileTask { c: KotlinJvmTestBuilder.TaskBuilder ->
+      c.setLabel("//deps")
       c.addSource(
         "AnotherClass.java",
         """
           package something;
           
           class AnotherClass {
           }
-        """
+        """,
       )
     }
     val jdeps = depsProto(deps)
-
-    assertThat(jdeps.dependencyCount).isEqualTo(0)
-    assertThat(jdeps.ruleLabel).isEqualTo(deps.label())
+    val expected = Deps.Dependencies.newBuilder()
+      .setRuleLabel("//deps")
+      .build()
+    assertThat(jdeps).isEqualTo(expected)
   }
 
   @Test
   fun `no dependencies`() {
-
     val deps = runJdepsCompileTask { c: KotlinJvmTestBuilder.TaskBuilder ->
+      c.setLabel("//deps")
       c.addDirectDependencies(TEST_FIXTURES_DEP)
     }
-    val jdeps = depsProto(deps)
 
-    assertThat(jdeps.dependencyCount).isEqualTo(0)
-    assertThat(jdeps.ruleLabel).isEqualTo(deps.label())
+    val jdeps = depsProto(deps)
+    val expected = Deps.Dependencies.newBuilder()
+      .setRuleLabel("//deps")
+      .build()
+    assertThat(jdeps).isEqualTo(expected)
   }
 
   @Test
   fun `java class static reference`() {
     val dependingTarget = runJdepsCompileTask { c: KotlinJvmTestBuilder.TaskBuilder ->
+      c.setLabel("//dependingTarget")
       c.addSource(
         "AClass.kt",
         """
           package something
           
           val result = JavaClass.staticMethod()
-        """
+        """,
       )
       c.addDirectDependencies(TEST_FIXTURES_DEP)
     }
-    val jdeps = depsProto(dependingTarget)
 
-    assertExplicit(jdeps).containsExactly(TEST_FIXTURES_DEP.singleCompileJar())
-    assertImplicit(jdeps).isEmpty()
-    assertUnused(jdeps).isEmpty()
-    assertIncomplete(jdeps).isEmpty()
+    val jdeps = depsProto(dependingTarget)
```

Comment:
To de-risk this PR I left the original assertions in place until I finished making all of the new assertions pass.